src/test/incremental/hello_world.rs

```diff
@@ -18,22 +18,23 @@ fn main() { }
 
 mod x {
     #[cfg(rpass1)]
-    pub fn x() -> i32 {
+    pub fn xxxx() -> i32 {
         1
     }
 
     #[cfg(rpass2)]
-    pub fn x() -> i32 {
+    pub fn xxxx() -> i32 {
         2
     }
 }
 
 mod y {
     use x;
 
-    #[rustc_clean(label="TypeckItemBody", cfg="rpass2")]
-    pub fn y() {
-        x::x();
+    // FIXME: This should be clean
```

Comment:
I think the problem is the issue of collect reading from the fn body that I pointed out earlier.